libs/api/src/main/java/de/cyface/api/MeasurementRetrievalWithSensorData.java

```diff
@@ -67,6 +72,8 @@ public TrackBucket trackBucket(final JsonObject document) throws ParseException
         final var directions = new ArrayList<>(point3D(directionsDocuments));
 
         final var track = new Track(locationRecords, accelerations, rotations, directions);
-        return new TrackBucket(trackId, bucket, track, metaData);
+        // noinspection SpellCheckingInspection
+        final var date = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").parse(bucket.getString("$date"));
+        return new TrackBucket(trackId, date, track, metaData);
```

Comment:
I have to unpack the `JsonObject` to `Date` now here, as the `serialization/libs/model` has no dependency to `Vertx.JsonObject`.